README.md

```diff
@@ -516,14 +517,15 @@ val jsJodaTz = JsJodaTimeZoneModule
 
 Wasm/JS uses the same time zone support as JS, so almost the same instructions apply.
 
-In your Gradle build script, add the following dependency:
+In your Gradle build script, add the following dependency
+([check here](https://www.npmjs.com/package/@js-joda/timezone?activeTab=versions) for the latest version):
```

Comment:
Please also explicitly mention that the user needs to replace `X.X.X` with that version, to streamline the process of blindly copy-pasting this snippet.